internal/packages/internal/packageimport/fs.go

```diff
@@ -40,7 +96,17 @@ func FromFS(ctx context.Context, src fs.FS) (*packagetypes.RawPackage, error) {
 		case entry.IsDir():
 			// no special handling for directories
 
+		case entry.Type()&os.ModeSymlink == os.ModeSymlink:
+			// symlink
+			finalPath, err := src.Readlink(path)
+			if err != nil {
+				return err
+			}
+			_, err = os.Lstat(finalPath)
```

Comment:
this Lstat should not run against the host os but against the wrapped fs instance, correct?

Reply:
no longer applies